pkg/operator/platform/platform.go

```diff
@@ -89,12 +93,22 @@ func GetFeatureGates() (featuregates.FeatureGate, error) {
 	desiredVersion := computeClusterOperatorVersions()
 	missingVersion := desiredVersion
 
+	kubeClient, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		return nil, fmt.Errorf("failed to create kube client: %w", err)
+	}
+	eventRecorder := events.NewKubeRecorder(kubeClient.CoreV1().Events("openshift-cloud-credential-operator"), "cloud-credential-operator", &corev1.ObjectReference{
+		APIVersion: "apps/v1",
+		Kind:       "Deployment",
+		Namespace:  "openshift-cloud-credential-operator",
+		Name:       "cloud-credential-operator",
```

Comment:
I hardcoded values for namespace & deployment here but they should probably be constants.

Reply:
moved to constants in upcoming commit